tsl/test/sql/transparent_decompression_queries.sql

```diff
@@ -67,3 +67,22 @@ GROUP BY 2, 3;
 
 RESET enable_hashagg;
 
+-- test if volatile function quals are applied to compressed chunks
+CREATE FUNCTION check_equal_228( intval INTEGER) RETURNS BOOL 
+LANGUAGE PLPGSQL AS
```

Comment:
I know that functions are volatile by default, but it might make sense to make it explicit given that this is what we test for.

```suggestion
LANGUAGE PLPGSQL VOLATILE AS
```